### PR TITLE
Support Time fields, using datetime.datetime

### PR DESCRIPTION
**Pull request by mortenlj**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 GENERIC_REQ = [
     "six == 1.10.0",
-    "requests == 2.13.0"
+    "requests == 2.13.0",
+    "pyrfc3339 == 1.0"
```
Comment (oyvindio) [inline setup.py]: nit: adding a trailing comma on the last list item makes diffs like this (very slightly) more readable

**Comment by mortenlj** [comment]
> LGTM, assuming rfc3339 is the format Kubernetes wants/emits

https://v1-7.docs.kubernetes.io/docs/api-reference/v1.7/#objectmeta-v1-meta

In the description of the fields using the `Time` type, they say RFC3339 using UTC. Strangely enough, if you look at the `Time` type itself, they don't say anything.. :disappointed:

File: docs/source/modules/k8s.models.configmap.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+k8s\.models\.configmap module
```
Comment (mortenlj) [inline docs/source/modules/k8s.models.configmap.rst]: These files seem to be autogenerated, should they perhaps not be checked in, and rather be generated when building docs?

Comment (oyvindio) [reply docs/source/modules/k8s.models.configmap.rst]: If readthedocs can generate it, then I think we should let it

Comment (mortenlj) [reply docs/source/modules/k8s.models.configmap.rst]: They are generated by `make html` at least, but I don't know what readthedocs uses.. Anyway, we can handle that in a different PR.

**Pull request merged**